<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #B2B1CF;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="interest_rate_2">
       <property name="styleSheet">
        <string notr="true">color: black</string>
       </property>
       <property name="text">
        <string>Способ погашения:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_annuity">
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Аннуитетный</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_differentiated">
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Дифференцированный</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="Monthly_pay">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Ежемесячная оплата</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Monthly_pay_sum">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Monthly_pay_rub">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>рублей</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="credit_term">
       <property name="styleSheet">
        <string notr="true">color: black; font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Срок кредита:          </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="credit_term_pay">
       <property name="styleSheet">
        <string notr="true">color: black; background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="years">
       <property name="styleSheet">
        <string notr="true">color: black; font-size: 15px</string>
       </property>
       <property name="text">
        <string>лет</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #EA526F;
color: black;</string>
       </property>
       <property name="text">
        <string>Рассчет</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="interest_rate">
       <property name="styleSheet">
        <string notr="true">color: black; font-size: 15px;</string>
       </property>
       <property name="text">
        <string>Процентная ставка:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="percent_rate">
       <property name="styleSheet">
        <string notr="true">color: black; background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="percent">
       <property name="styleSheet">
        <string notr="true">color: black; font-size: 15px</string>
       </property>
       <property name="text">
        <string>   %</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="Overpayment_pay">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Переплата</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Overpayment_sum">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Overpayment_rub">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>рублей</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="Total_payment">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>Сумма</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="total_payment_sum">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="total_payment_rub">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="text">
        <string>рублей</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="tota_loan_label">
       <property name="styleSheet">
        <string notr="true">color: black; font-size: 15px</string>
       </property>
       <property name="text">
        <string>Сумма кредита:       </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="totalamount">
       <property name="styleSheet">
        <string notr="true">color: black;
background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rub">
       <property name="styleSheet">
        <string notr="true">color: black; font-size: 15px</string>
       </property>
       <property name="text">
        <string>    ₽</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
